### ISSUE TITLE
[CARKEYS EVENT?] Help me please

### ISSUE THREAD

**Issue by Gameadictive** [renzuzu/renzu_shops#116]
Hi, good renzu, I have a question. 

I'm using Jaksam's jobs_creator and I would like to make it so that when they buy a car with their script, they give me the keys to open and close it as in your script, which works perfectly. I am using ESX and looking for the event that calls the action, could you help me?

**Comment by renzuzu** [comment]
car keys event in init.lua

**Comment by Gameadictive** [comment]
Yes yes but i need help to set it to jaksams jobs_creator if possible. They dont give support to edit code cause is a paid code. They have documentation but i dont understand how to.

**Comment by Gameadictive** [comment]
![cl_integrations](https://user-images.githubusercontent.com/65845786/231440332-2c3b749f-ca6a-4f09-95d8-bc9d105d3a30.PNG)
![sh_integrations](https://user-images.githubusercontent.com/65845786/231440334-55a9a3bd-a8c1-4826-b531-ce85e472e7a3.PNG)
![sv_integrations](https://user-images.githubusercontent.com/65845786/231440337-b42006dd-2031-46e0-935b-51809632a088.PNG)

**Comment by Gameadictive** [comment]
If u can help me would be awesome renzu.

https://documentation.jaksam-scripts.com/jaksams-scripts/home

**Comment by renzuzu** [comment]
the confusion is, ddo you have key resource?
this resource is shop/ which triggered key events too
